pdf/model/fonts/ttfparser.go

```diff
@@ -83,13 +84,32 @@ type TtfType struct {
 	CapHeight              int16
 	Widths                 []uint16
 
-	// Chars maps rune values (unicode) to the indexes in GlyphNames. i.e GlyphNames[Chars[r]] is
+	// Chars maps rune values (unicode) to the indexes in GlyphNames. i.e. GlyphNames[Chars[r]] is
 	// the glyph corresponding to rune r.
 	Chars map[uint16]uint16
 	// GlyphNames is a list of glyphs from the "post" section of the TrueType file.
 	GlyphNames []string
 }
 
+// MakeToUnicode returns a ToUnicode CMap based on the encoding of `ttf`.
+// XX: This currently gives a bad text mapping for creator_test.go but leads to an otherwise
```

Comment:
Use XXX(author).
We need to make a test case for this going forward.  Will make a ticket for it.